pdk/1.0/modules/ROOT/pages/policies-pdk-integration-tests.adoc

```diff
@@ -6,14 +6,19 @@ endif::[]
 
 Flex Gateway Policy Development Kit (PDK) provides an automated testing framework to detect integration regressions in your custom policy.
 
+The integration testing framework provided by PDK only supports Flex Gateways running in Local Mode in a Docker container. If you plan to deploy or policy to a Flex Gateway running in Connected Mode or to a different platform, you should still test your policy in PDK by using Flex Gateways running in Local Mode in a Docker container for initial testing.
```

Comment:
```suggestion
The integration testing framework provided by PDK only supports Flex Gateway running in Local Mode in a Docker container. If you plan to deploy or policy to Flex Gateway running in Connected Mode or to a different platform, you should still first test your policy in PDK by using Flex Gateway running in Local Mode in a Docker container.
```
Same question about "or to a different platform". Is this referring to a production environment?